apps/generate.nix

```diff
@@ -56,7 +56,9 @@
   # If the path already exists, this makes sure that the definition is the same.
   addGeneratedSecretChecked = host: set: secretName: let
     secret = nodes.${host}.config.age.secrets.${secretName};
-    sourceFile = relativeToFlake secret.rekeyFile;
+    sourceFile = assert assertMsg (secret.rekeyFile != null)
+    "age.secrets.${secretName}: `rekeyFile` must be set when using a generator.";
```

Comment:
Since we have information about the host, can we add it to the message? Like `host ${host}: age.secrets.[...]`